samwise.js

```diff
@@ -5,20 +5,23 @@
  * Released under the MIT license.
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.